pkgs/development/tools/conftest/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+let version = "0.7.0"; in
+
+buildGoModule {
+  pname = "helmfile";
+  inherit version;
```

Comment:
Please inline the version here (remove the whole let statement. You need to use`rec` obviously.

Reply:
I tend to prefer to avoid `rec` when trivial, as it can lead to ambiguity. In this case I don't have strong enough feelings, so I'll take your suggestion.